### PR TITLE
Editable-Form

### PR DESCRIPTION
**Pull request by ITsunilthapa**
Created two fields for start date and end date on the form

### CODE REVIEW

File: packages/form/src/components/Action.vue [hunk 1]
```diff
@@ -67,7 +99,9 @@
               action: {
                 name        : { [locale]: '' },
                 description : { [locale]: '' },
-                attachments : []
+                attachments : [],
+                startdate   : { [locale]: '' },
```
Comment (randyhoulahan) [inline packages/form/src/components/Action.vue:103]: we always want to use camelCase in JS

File: packages/form/src/components/Action.vue [hunk 2]
```diff
@@ -36,11 +36,43 @@
...
+
+    <div class="col-12">
+      <div class="form-group" id="group.action.enddate">
+        <label  for="action.enddate"> {{ $t(`End Date`) }} </label>
+        <input class="form-control" 
+          @input      ="update"
+          id          ="action.enddate"
+          type        ="date"
+          v-model.trim="action.enddate[$i18n.locale]"
+          v-validate  ="'required|max:140'"
```
Comment (randyhoulahan) [inline packages/form/src/components/Action.vue:68]: max:140 - is a length of string.  Not needed for date.  What validation does the v-validate library have for dates? https://vee-validate.logaretm.com/v2/guide/rules.html#date-format.  Reference the package.json, we use version 2.2.15 here.  Not the latest version is 4.

File: packages/form/src/components/Action.vue [hunk 3]
```diff
@@ -36,11 +36,43 @@
     </div>
     <div class="col-12">
       <div class="form-group" id="group.action.attachments">
-        <label :for="`action.attachments`"> {{ $t(`Attachement (s)`) }} </label>
+        <label :for="`action.attachments`"> {{ $t(`Attachment (s)`) }} </label>
```
Comment (randyhoulahan) [inline packages/form/src/components/Action.vue:39]: thank you, my spelling is horrendous.

**Review by randyhoulahan — CHANGES_REQUESTED** [review]
Thank you.  I think it may be nicer to have both properties on one row or on one row occupying the first 2/3's of the row.  The user will see the little date icon to select a date easier.  Please note how the data is formed in the debug box and ultimately sent to the server.    Instead of the LString object just a string property.  We may need to send it as { $date: '2021-07-27' } so mongo store the property as a date.  I will update the api and confirm.

I cleaned the branch and moved your work here to this branch https://github.com/scbd/action-agenda-components/tree/v1-form-date-range-props.  Please update on that branch.


It looks like the default html date select is fine and a separate component is not needed.

Screen Shot 2021-07-27 at 9 03 34 AM


Screen Shot 2021-07-27 at 9 05 18 AM

**Comment by ITsunilthapa** [comment]
This branch will be closed as work has been moved to the new branch: https://github.com/scbd/action-agenda-components/tree/v1-form-date-range-props.

**Pull request closed without merging**